config/locales/client.en.yml

```diff
@@ -3043,7 +3043,7 @@ en:
       in_messages_tooltip: "switch to searching regular topics"
       in_topics_posts: "in all topics and posts"
       enter_hint: "or press Enter"
-      mobile_enter_hint: "or tap Search icon"
+      mobile_enter_hint: "or tap Search"
```

Comment:
`tap Search` reads ambiguous to me, but am not insisting on keeping `icon`

Reply:
It's because we recently implemented the default keyboard functionality of displaying "search" which is not an icon. Technially, the OG icon still works too (so there is 3 ways to trigger the search on mobile) but it's far less intuitive than keyboard search
![CleanShot 2025-04-23 at 13 41 21](https://github.com/user-attachments/assets/a4423ca1-d7f7-4ba9-875e-19cd70b9b0f0)

Reply:
I meant this icon 
<img width="611" alt="Screenshot 2025-04-24 at 19 38 35" src="https://github.com/user-attachments/assets/660ac3d7-d774-496b-8149-3cca8b3ba9b8" />

Reply:
hence I said:  _Technially, the OG icon still works too (so there is 3 ways to trigger the search on mobile) but it's far less intuitive than keyboard search_